content/1.7/back-office/catalog/discount/add-edit-cart-price-rule.md

```diff
@@ -0,0 +1,79 @@
+- **Description** - it can be described where is the cart rule used for in more details.
+- **Code** - unique cart rule code can be set here, or simply by clicking Generate and Prestashop will generate the rule automatically with unique code name. It will become as voucher from front end side.
+- **Highlight switch toggle button** - this option can be enabled, then the cart rule voucher will be displayed in cart summary. When there is no Voucher Code set, the Highlight switch toggle button is hidden.
+- **Partial use switch toggle button** - this option can be enabled, when the total cart sum is smaller than the voucher sum. If Partial usage is disabled, voucher value will be lovered to the total order amount. If it will be enabled, new voucher will be created in the Cart, as a reminder. Partial use toggle button is switched on by default.
+- **Priority** - priority of Cart rules can be set here, in what order to show the rules for the customers of the webshop. For example, setting 1 for the most importand cart rule. Priority input field is stated with digital value 1 by default. If there are multiple cart rules created, with the identical digital value, Prestashop Back-Office will arrange the cart rules like the Cart Rule ID arrangement is set in module Cart Rule listing - from the older to the newest created.
+- **Status switch toggle button** - enabling this button will enable the rule, disabling it will deactivate the rule and vouchers for customers. Status toggle button is enabled by default.
+
+## Conditions tab
+The condition tab, allows the ability to specify cart rule conditions to determine how the cart rule will be used in the Prestashop. 
+
+[Cart Rules: Conditions Tab](/static/img/cart-rules-conditions-tab.png)
+
+- **Limit to a single customer input field** - this field sets the customer name or email in order to set the Cart rule condition for certain customer.
+- **Valid** - simple date picker form, where date existance can be set. Valid data picker widget is stated for the 1 month value by default.
+- **Minimum amount field** - this field sets minimum amount from which the customers will see the Cart rule. As well, the currency can be set, if prices should affected with taxes or without taxes and if shipping is affected too. Minimum amount is stated by digital value 0. Default currency is stated by the default Prestashop currency. Tax excluded dropdown is stated by default. Shipping excluded dropdown is stated by default.
+- **Total available** - this filed describes the quantity of the cart rules - vouchers available for certain conditions. Total available input field is stated by digital value 1 by default.
+- **Total available for each user field** - this field sets the price rules amount for certain users separately. Total available for each user is stated by digital value 1 by default.
```

Comment:
What happens if the total available is higher than the total available for each customer? Is the customer can still use the cart rule after using all the total available  for each customer even though there is some total available left?
Can the user still use the cart rule is the total availability is lower than Total available for each user?

Reply:
@MatShir I have checked, that the error is thrown in that case "This voucher has already been used."